app/Models/Course.php

```diff
@@ -52,6 +56,18 @@ public function getRomanizedSemester(): string
         };
     }
 
+    public function getSemesterName(): string
+    {
+        return $this->semester % 2 === 0
+            ? __("Letni")
```

Comment:
I don't think we have any translations anywhere.

Reply:
fixed